src/ChannelServer/Network/Sending/Send.Parties.cs

```diff
@@ -0,0 +1,447 @@
+﻿// Copyright (c) Aura development team - Licensed under GNU GPL
+// For more information, see license file in the main folder
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Aura.Channel.World.Entities;
+using Aura.Shared.Network;
+using Aura.Channel.World;
+using Aura.Channel.Skills;
+using Aura.Mabi.Const;
+using Aura.Mabi.Network;
+
+namespace Aura.Channel.Network.Sending
+{
+    public static partial class Send
+    {
+        /// <summary>
+        /// Response to the party creation request, sends the client the relevant party data.
+        /// </summary>
+        /// <remarks>I feel like I'm the MSDN with that summary.</remarks>
+        /// <param name="creature"></param>
+        public static void CreatePartyR(Creature creature)
+        {
+            var packet = new Packet(Op.PartyCreateR, creature.EntityId);
+
+            var party = creature.Party;
+
+            if (creature.IsInParty)
+            {
+                packet.PutByte(1);
+
+                BuildPartyInfo(party, ref packet);
+            }
+            else
+                packet.PutByte(0);
+
+            creature.Client.Send(packet);
+        }
+
+        /// <summary>
+        /// Sends the correct response to the client sending a party join request,
+        /// and success grants full party information.
+        /// </summary>
+        /// <param name="creature"></param>
+        /// <param name="result"></param>
+        public static void PartyJoinR(Creature creature, PartyJoinResult result)
+        {
+            var packet = new Packet(Op.PartyJoinR, creature.EntityId);
+
+            if (PartyJoinResult.WrongPass == result || PartyJoinResult.Full == result)
+            {
+                packet.PutByte((byte)result);
+                creature.Client.Send(packet);
+                return;
```

Comment:
It would appear that *HOSTING* one is illegal, but developing one seems to be fine.
If they had an issue with it, why would they not have had all the repos destroyed?